### PR TITLE
fix(playlist)/this-icon-url-is-not-function

### PR DESCRIPTION
**Pull request by vugar005**
Thanks for this awesome project.
This PR fixes issue: this iconUrl is not function
Reproducing steps: 
Try: YouTube.getPlaylist("PL9NhZUhtpyDasHTIDoLBhQk3Y113WfmGo")

### CODE REVIEW

File: deno/Structures/Playlist.ts [hunk 1]
```diff
@@ -141,7 +141,7 @@ class Playlist {
             channel: {
                 name: this.channel.name,
                 id: this.channel.id,
-                icon: this.channel.iconURL()
+                icon: this.channel?.iconURL ? this.channel.iconURL(): null
```
Comment (DevAndromeda) [inline deno/Structures/Playlist.ts:144]:
```suggestion
                icon: this.channel?.iconURL?.()
```

File: src/Structures/Playlist.ts [hunk 2]
```diff
@@ -140,7 +140,7 @@ class Playlist {
             channel: {
                 name: this.channel.name,
                 id: this.channel.id,
-                icon: this.channel.iconURL()
+                icon: this.channel?.iconURL ? this.channel.iconURL(): null
```
Comment (DevAndromeda) [inline src/Structures/Playlist.ts:143]:
```suggestion
                icon: this.channel?.iconURL?.()
```

**Review by DevAndromeda — CHANGES_REQUESTED** [review]
let's do it this way instead?

**Comment by vugar005** [comment]
@DevAndromeda sounds good! Applied suggestions)

**Review by DevAndromeda — APPROVED** [review]

**Pull request merged**